utils/auth.js

```diff
@@ -0,0 +1,31 @@
+import { Popup, popupWindowOptions } from '@/utils/window';
+
+const PROVIDER_CALLBACK_MAPPING = { azuread: 'onAzureTest' };

```

Comment:
If it weren't for verify-auth-azure crossing the streams and loading an ember page, this seems like it could have worked the same as github did in `store/auth test()`?

Reply:
You were right. After we stopped crossing streams this could be removed.